ouroboros-network/src/Ouroboros/Network/PeerSelection/Churn.hs

```diff
@@ -79,7 +79,9 @@ peerChurnGovernor tracer deadlineChurnInterval bulkChurnInterval
                    ChurnModeNormal  ->
                        targetNumberOfActivePeers base
                    ChurnModeBulkSync ->
-                       min 2 (targetNumberOfActivePeers base)
+                       min 2 (targetNumberOfActivePeers base),
+          targetNumberOfActiveBigLedgerPeers =
+              targetNumberOfActiveBigLedgerPeers base
```

Comment:
until we have chain-sync skipping we should limit the number of active big ledger peers in `ChurnModeBulkSync`.

Reply:
input-output-hk/ouroboros-network#4609